src/python/grpcio/grpc/_channel.py

```diff
@@ -1224,12 +1230,14 @@ def __init__(
         method: bytes,
         request_serializer: SerializingFunction,
         response_deserializer: DeserializingFunction,
+        registered_method: bool,
```

Comment:
This parameter name should start with an underscore to indicate that it's for internal usage only. We probably also want to add a docstring mentioning its purpose.